packages/theme/nuxt.config.js

```diff
@@ -3,200 +3,210 @@ import theme from './themeConfig';
 import webpack from 'webpack';
 import * as dotenv from 'dotenv';
 import urlJoin from 'url-join';
+import getStore from './getStore';
 
 dotenv.config({ path: `./../../.env.${process.env.NODE_ENV}` });
 
 const serverConfig = {
   port: process.env.PORT || 3000,
   host: process.env.HOST || '0.0.0.0',
-  baseUrl: process.env.BASE_URL || 'http://localhost:3000'
+  baseUrl: process.env.BASE_URL || 'http://localhost:3000',
+  backendUrl: process.env.BACKEND_URL,
+  storeApiRoute: process.env.STORE_API_ROUTE || '/api/v2/storefront/store'
```

Comment:
Yeah, I think it's cleaner to inject everything from the serverConfig. Good change.